cli/script/command-executor.ts

```diff
@@ -657,7 +739,69 @@ function printDeploymentHistory(command: cli.IDeploymentHistoryCommand, packageH
                     releaseTime,
                     packageObject.appVersion,
                     packageObject.isMandatory ? "Yes" : "No",
-                    packageObject.description ? wordwrap(30)(packageObject.description) : ""
+                    packageObject.description ? wordwrap(30)(packageObject.description) : "",
+                    getPackageMetricsString(packageObject)
+                ]);
+            });
+        });
+    }
+}
+
+function printDeploymentMetrics(command: cli.IDeploymentMetricsCommand, metrics: any): void {
+    var labelToMetricsMap: { [label: string] : DeploymentMetric } = {};
+    var totalActive: number = 0;
+    var versionRegex = /v(\d+)/;
+    
+    Object.keys(metrics).forEach((metricKey: string) => {
+        var parsedKey: string[] = metricKey.split(":");
```

Comment:
Or another option is we could move this parsing into the SDK. This makes it less friendly to interact with the API directly, but it means less processing on the server. I'd still lean towards server, but you might have different ideas on the trade-offs (if you agree with me on this point in the first place that is).